### ISSUE TITLE
Improve representation of nitrogen cycle metabolic process

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#27217]
This is an epic ticket

This:

* [] GO:0006807 ! nitrogen compound metabolic process
    * [i] **GO:0071941 ! nitrogen cycle metabolic process**
        * [i] GO:0042128 ! nitrate assimilation
        * [i] GO:0019627 ! urea metabolic process
            * [i] GO:0043419 ! urea catabolic process
            * [i] GO:0019623 ! atrazine catabolic process to urea
            * [i] GO:0000050 ! urea cycle
        * [i] GO:0019333 ! denitrification pathway
        * [i] GO:0009399 ! nitrogen fixation

Should look more like this:

<img width="1031" alt="image" src="https://github.com/geneontology/go-ontology/assets/50745/1578b533-aaed-47a8-98b1-f962efffdb1f">

- [x] #27216.  
- [ ] #27215.
- [x] #27210 
    - fix: #27213.
- [ ] #27220
- [ ] #27221

**Comment by deustp01** [comment]
Probably a tangent, but ...

The definition of GO:0071941 nitrogen cycle metabolic process is "The nitrogen cycle is a series of metabolic pathways by which nitrogen is converted between various forms and redox states; it encompasses pathways in which nitrogen is acted upon directly, such as nitrification, denitrification, nitrogen fixation, and mineralization. PMID:16675690"

Is this intended to apply potentially to all taxa or to be restricted to prokaryotes and plants? Does the clause "in which nitrogen is acted upon directly" apply to all nitrogen cycle processes, or is it an optional label for some of them? If the former (all), then urea cycle is out of place because it does not involve conversion of nitrogen the element to anything. If the latter (optional), should purine catabolism to uric acid be added as a sibling of urea cycle, as playing a role in birds and reptiles like the one for urea cycle in mammals?

**Comment by cmungall** [comment]
I agree @deustp01 - made an issue for this, #27220, and added to the epic checklist above

**Comment by ValWood** [comment]
Note, I have seen the urea cycle referred to biologically as part of the "nitrogen cycle". Note that the ecological meaning of "nitrogen cycle" is probably different.  

This term was intended to group process like nitrogen fixation and urea cycle that act on NH4 and NH3. If we move this term out we need a way to group terms  but in that case it should also include "GO:0019676 ammonia assimilation cycle" or GO:0019740  nitrogen utilization

Note  this is narrower  than the parent "organonitrogen compound metabolic process (GO:1901564)"  or "GO:0006807  nitrogen compound metabolic process"
(which seem to be a meaningless grouping term including many, but not all things that 'contain' any nitrogen' currently 2/5 of the pombe proteome are annotated to these terms).

Here is the ticket where the parent was added:
https://github.com/geneontology/go-ontology/issues/8040

TLDR; I think this term was intended to mean "nitrogen metabolic process" but I seem to remember that term was not approved of because it isn't "nitrogen" that is acted on in vivo (I cannot locate this ticket, it's probably from source forge days).

**Comment by ValWood** [comment]
in case useful https://www.pombase.org/gocam/gene/665912ed00002626/SPAC1F7.09c/dal2

**Comment by ValWood** [comment]
these are the correct members
https://www.pombase.org/term/GO:0042272

**Comment by raymond91125** [comment]
@ValWood Did you mean to close this ticket?

**Comment by ValWood** [comment]
That was a mistake!

**Comment by raymond91125** [comment]
All issues have been dealt with.
